<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SAWin11Dev" failures="0" tests="16" name="com.apex.samples.UserAPITestNGGetXLS" time="12.394" errors="5" timestamp="2022-03-02T12:17:56 PST" skipped="0">
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithLowerCaseLetters" time="0.992"/>
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithNegativeNumber" time="0.996"/>
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithAlphaNumerics" time="2.473"/>
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithWrongDataType" time="0.982"/>
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithNumber" time="0.990"/>
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithMixOfUpperNLowerCaseLetters" time="0.988"/>
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithSuccessData" time="0.997"/>
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithBlankInFirstPosition" time="0.002">
    <error message="Illegal character in path at index 38: https://gorest.co.in/public-api/users/ 1383" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: Illegal character in path at index 38: https://gorest.co.in/public-api/users/ 1383
at java.base/java.net.URI.create(URI.java:906)
at org.apache.http.client.methods.HttpGet.<init>(HttpGet.java:66)
at com.apex.samples.UserAPITestNGGetXLS.sendGetRequest(UserAPITestNGGetXLS.java:30)
at com.apex.samples.UserAPITestNGGetXLS.testGetwithBlankInFirstPosition(UserAPITestNGGetXLS.java:189)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.URISyntaxException: Illegal character in path at index 38: https://gorest.co.in/public-api/users/ 1383
at java.base/java.net.URI$Parser.fail(URI.java:2974)
at java.base/java.net.URI$Parser.checkChars(URI.java:3145)
at java.base/java.net.URI$Parser.parseHierarchical(URI.java:3227)
at java.base/java.net.URI$Parser.parse(URI.java:3175)
at java.base/java.net.URI.<init>(URI.java:623)
at java.base/java.net.URI.create(URI.java:904)
... 31 more
]]>
    </error>
  </testcase> <!-- testGetwithBlankInFirstPosition -->
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithSpecialCharacters" time="0.001">
    <error message="Malformed escape pair at index 41: https://gorest.co.in/public-api/users/@#$%^^&amp;" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: Malformed escape pair at index 41: https://gorest.co.in/public-api/users/@#$%^^&
at java.base/java.net.URI.create(URI.java:906)
at org.apache.http.client.methods.HttpGet.<init>(HttpGet.java:66)
at com.apex.samples.UserAPITestNGGetXLS.sendGetRequest(UserAPITestNGGetXLS.java:30)
at com.apex.samples.UserAPITestNGGetXLS.testGetwithSpecialCharacters(UserAPITestNGGetXLS.java:177)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.URISyntaxException: Malformed escape pair at index 41: https://gorest.co.in/public-api/users/@#$%^^&
at java.base/java.net.URI$Parser.fail(URI.java:2974)
at java.base/java.net.URI$Parser.scanEscape(URI.java:3102)
at java.base/java.net.URI$Parser.scan(URI.java:3125)
at java.base/java.net.URI$Parser.checkChars(URI.java:3143)
at java.base/java.net.URI$Parser.parse(URI.java:3189)
at java.base/java.net.URI.<init>(URI.java:623)
at java.base/java.net.URI.create(URI.java:904)
... 31 more
]]>
    </error>
  </testcase> <!-- testGetwithSpecialCharacters -->
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithUpperCaseLetters" time="0.972"/>
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithZero" time="0.993"/>
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithHTMLCharacters" time="0.003">
    <error message="Illegal character in path at index 38: https://gorest.co.in/public-api/users/&lt;!$#&gt;" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: Illegal character in path at index 38: https://gorest.co.in/public-api/users/<!$#>
at java.base/java.net.URI.create(URI.java:906)
at org.apache.http.client.methods.HttpGet.<init>(HttpGet.java:66)
at com.apex.samples.UserAPITestNGGetXLS.sendGetRequest(UserAPITestNGGetXLS.java:30)
at com.apex.samples.UserAPITestNGGetXLS.testGetwithHTMLCharacters(UserAPITestNGGetXLS.java:261)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.URISyntaxException: Illegal character in path at index 38: https://gorest.co.in/public-api/users/<!$#>
at java.base/java.net.URI$Parser.fail(URI.java:2974)
at java.base/java.net.URI$Parser.checkChars(URI.java:3145)
at java.base/java.net.URI$Parser.parseHierarchical(URI.java:3227)
at java.base/java.net.URI$Parser.parse(URI.java:3175)
at java.base/java.net.URI.<init>(URI.java:623)
at java.base/java.net.URI.create(URI.java:904)
... 31 more
]]>
    </error>
  </testcase> <!-- testGetwithHTMLCharacters -->
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithSuccessData" time="1.004"/>
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithCharacters" time="0.996"/>
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithBlank" time="0.003">
    <error message="Illegal character in path at index 38: https://gorest.co.in/public-api/users/ " type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: Illegal character in path at index 38: https://gorest.co.in/public-api/users/ 
at java.base/java.net.URI.create(URI.java:906)
at org.apache.http.client.methods.HttpGet.<init>(HttpGet.java:66)
at com.apex.samples.UserAPITestNGGetXLS.sendGetRequest(UserAPITestNGGetXLS.java:30)
at com.apex.samples.UserAPITestNGGetXLS.testGetwithBlank(UserAPITestNGGetXLS.java:102)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.URISyntaxException: Illegal character in path at index 38: https://gorest.co.in/public-api/users/ 
at java.base/java.net.URI$Parser.fail(URI.java:2974)
at java.base/java.net.URI$Parser.checkChars(URI.java:3145)
at java.base/java.net.URI$Parser.parseHierarchical(URI.java:3227)
at java.base/java.net.URI$Parser.parse(URI.java:3175)
at java.base/java.net.URI.<init>(URI.java:623)
at java.base/java.net.URI.create(URI.java:904)
... 31 more
]]>
    </error>
  </testcase> <!-- testGetwithBlank -->
  <system-out/>
  <testcase classname="com.apex.samples.UserAPITestNGGetXLS" name="testGetwithBlankInLastPosition" time="0.002">
    <error message="Illegal character in path at index 42: https://gorest.co.in/public-api/users/1383 " type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: Illegal character in path at index 42: https://gorest.co.in/public-api/users/1383 
at java.base/java.net.URI.create(URI.java:906)
at org.apache.http.client.methods.HttpGet.<init>(HttpGet.java:66)
at com.apex.samples.UserAPITestNGGetXLS.sendGetRequest(UserAPITestNGGetXLS.java:30)
at com.apex.samples.UserAPITestNGGetXLS.testGetwithBlankInLastPosition(UserAPITestNGGetXLS.java:201)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.URISyntaxException: Illegal character in path at index 42: https://gorest.co.in/public-api/users/1383 
at java.base/java.net.URI$Parser.fail(URI.java:2974)
at java.base/java.net.URI$Parser.checkChars(URI.java:3145)
at java.base/java.net.URI$Parser.parseHierarchical(URI.java:3227)
at java.base/java.net.URI$Parser.parse(URI.java:3175)
at java.base/java.net.URI.<init>(URI.java:623)
at java.base/java.net.URI.create(URI.java:904)
... 31 more
]]>
    </error>
  </testcase> <!-- testGetwithBlankInLastPosition -->
  <system-out/>
</testsuite> <!-- com.apex.samples.UserAPITestNGGetXLS -->
